core/include/gnuradio-4.0/BlockTraits.hpp

```diff
@@ -311,6 +311,12 @@ struct DummyConsumableSpan {
 };
 static_assert(ConsumableSpan<DummyConsumableSpan<int>>);
 
+template<typename T>
+struct DummyConsumablePortSpan: public DummyConsumableSpan<T> {
```

Comment:
The tags etc should be absorbed in the `concept ConsumableSpan` etc.

Reply:
I'm not sure I understand.
There are 2 concepts `ConsumableSpan`(no tag handling) and `ConsumablePortSpan`(the former + added tag handling). `DummyConsumableSpan` and `DummyConsumablePortSpan` are mock implementations of the respective concepts needed by the machinery of `Block::prepareStreams`.